src/databricks/labs/ucx/source_code/known.json

```diff
@@ -5149,6 +5149,24 @@
     "johnsnowlabs.visual": [],
     "johnsnowlabs.viz": []
   },
+  "jsonschema": {
+    "jsonschema": [],
+    "jsonschema._format": [],
+    "jsonschema._keywords": [],
+    "jsonschema._legacy_keywords": [],
+    "jsonschema._types": [],
+    "jsonschema._typing": [],
+    "jsonschema._utils": [],
+    "jsonschema.cli": [],
+    "jsonschema.exceptions": [],
+    "jsonschema.protocols": [],
+    "jsonschema.tests": [],
+    "jsonschema.validators": []
+  },
+  "jsonschema-specifications": {
```

Comment:
Looks like an internally used package.

Reply:
No it's not. Check https://pypi.org/project/jsonschema-specifications/